### DISCUSSION TITLE
Casadi 3.6.6 Ipopt running slower

### DISCUSSION THREAD

**Discussion by rtapia8484** [casadi/casadi#3817] [Q&A]
hello,
I recently switched from Casadi 3.6.5 to 3.6.6 and noticed that my optimization problem (solver ipopt) runs slower, from ~80ms to ~100ms. This was consistent when I switched back and forth between versions. Also noticed the same issue when using a different computer but much worse. Any clues as to why? 

Best,
Ricardo Tapia

**Comment by jgillis** [comment]
Can you check with the nigthly build? There was a compile flag regression bug in 3.6.6 which may be related. https://github.com/casadi/casadi/releases/tag/nightly-main

**Reply by rtapia8484** [reply]
Hello @jgillis,
Sorry, but I am a bit of a git noob. How would I go about checking this bug in the nightly build?

Best

**Reply by jgillis** [reply]
Just do `pip install <url to whl file>`.
e.g. `pip install https://github.com/casadi/casadi/releases/download/nightly-main/casadi-3.6.6.dev+main-cp312-none-macosx_11_0_arm64.whl` in the case of a mac OS and python 3.12

**Reply by rtapia8484** [reply]
Oh, using Matlab on windows and did not see a 3.6.6 version?

**Reply by jgillis** [reply]
In that case, just download and unzip casadi-main-windows64-matlab2018b.zip of the nigthly build link

**Comment by rtapia8484** [comment]
The nightly build is running on par with Casadi 3.6.5, tested on multiple computers. Not sure why that was happening. 

Thanks
